readthedocs/projects/views/private.py

```diff
@@ -726,7 +727,14 @@ def get_success_url(self):
 
 
 class DomainList(DomainMixin, ListViewWithForm):
-    pass
+    def get_context_data(self, **kwargs):
```

Comment:
I think it's better to put this code under `get` method, don't you think?

Reply:
I think that would be worse, actually. We need access to the `domain_list` from the context. The `get` method both calls `get_context_data` and consumes the context (but doesn't return it). As a result, we'd have to replace `get` and not merely call the super method and add functionality. Does that make sense?